pyproject.toml

```
@@ -37,7 +37,7 @@ keywords = ["pip", "requirements", "packaging"]
 dependencies = [
   # direct dependencies
   "build >= 1.0.0",
-  "click >= 8",
+  "click >= 8 , < 8.2",
```

I assume the whitespace before the comma is accidental?
```suggestion
  "click >= 8, < 8.2",
```

---

I was unsure about having a space there or not. If it looks like a mistake to you; let's apply the change. :+1: